src/classes/WalletConnectors/WalletConnectConnector.ts

```diff
@@ -119,6 +133,10 @@ export class WalletConnectConnector extends WalletConnector {
         await this.setChain(chainId)
     }
 
+    public async disconnect() {
+        await this.provider.disconnect()
```

Comment:
what happens if you disconnect a provider that isnt connected?

Reply:
Good point... If im not wrong, I think it throws an error